.github/workflows/main.yml

```diff
@@ -39,13 +48,22 @@ jobs:
           $(go env GOPATH)/bin/ziti-ci generate-build-info common/version/info_generated.go version
           $(go env GOPATH)/bin/gox -cgo -os=darwin -arch=amd64 -output=$GOX_OUTPUT ./...
 
```

Comment:
Can we skip building test artifacts on mac and windows, since they're probably not used in the smoketest?

Reply:
Sure can! Left those in to make new tests environments one less thing to think about but easy enough to add back later.